docs/medical-api/more-info/filters.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Filtering data"
+description: "How to filter data in Metriport."
+---
+
+Some features include the ability to filter data. In those cases, the syntax below can be used:
+- `unquoted text`: all words are required in to be present in the results, in any order.
```

Comment:
`all words are required in to be present in the results` it feels like were missing a word

Reply:
I don't understand what you mean. What would you suggest we change it to?